pkg/controller/elasticsearch/user/roles.go

```diff
@@ -18,12 +18,46 @@ const (
 	SuperUserBuiltinRole = "superuser"
 	// ProbeUserRole is the name of the role used by the internal probe user.
 	ProbeUserRole = "elastic_internal_probe_user"
+
+	// ApmUserRoleV6 is the name of the role used by the association user for 6.8.x APMServer instances.
```

Comment:
Why are we calling it the `association user`? Is it not just the user APM Server will use to connect to Elasticsearch?